tsconfig.json

```diff
@@ -1,3 +1,3 @@
 {
-    "compilerOptions": { "target": "esnext", "allowJs": true, "noEmit": true }
+    "compilerOptions": { "target": "esnext", "allowJs": true, "noEmit": true, "skipLibCheck": true }
```

Comment:
If you add this, `.d.ts` file won't be checked, and the only ts file we have is the `global.d.ts` so please don't add this

Reply:
I didn't notice :sweat_smile:. My bad.

I had to add that after `npm i --save-dev typescript` because `Iterator` collided:
```
node_modules/typescript/lib/lib.esnext.iterator.d.ts:147:9 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'Iterator' must be of type '{ new (): Iterator<never, void, void>; prototype: Iterator<never, void, void>; range(start: number, end: number, option: number | NumericRangeOptions<number>): IterableIterator<...>; range(start: bigint, end: number | bigint, option: bigint | NumericRangeOptions<...>): IterableIterator<...>; }', but here has type 'IteratorConstructor'.

147     var Iterator: IteratorConstructor;
            ~~~~~~~~

  global.d.ts:2:13
    2 declare var Iterator: {
                  ~~~~~~~~
    'Iterator' was also declared here.


Found 1 error in node_modules/typescript/lib/lib.esnext.iterator.d.ts:147
```

I'll uninstall TS